### ISSUE TITLE
Accessibility violation: Zooming and scaling must not be disabled

### ISSUE THREAD

**Issue by jessepearson** [woocommerce/storefront#919]
The below is copied from this forum thread:
https://wordpress.org/support/topic/accessibility-violation-zooming-and-scaling-must-not-be-disabled/

=========

This theme disables zooming and scale which is a WCAG 2.0 (AA) violation.

Being able to zoom is essential for users with low vision.

This page provides more information and the solution: https://dequeuniversity.com/rules/axe/3.0/meta-viewport?application=AxeChrome
